client/my-sites/plans/main.jsx

```diff
@@ -31,7 +31,7 @@ const Plans = React.createClass( {
 	getDefaultProps() {
 		return {
 			intervalType: 'yearly',
-			displayJetpackPlans: true
+			displayJetpackPlans: false
```

Comment:
@johnHackworth the problem is that we get both `isSiteAutomatedTransfer` and `jetpackSite` from the API. While we don't have them (ie plans data is loading), `displayJetpackPlans` would be `undefined` and might cause issues. That's my main reason for having a default for something which is loaded dynamically (ie from some remote server).